rueidisotel/trace.go

```diff
@@ -83,6 +99,10 @@ func (o *otelclient) DoMulti(ctx context.Context, multi ...rueidis.Completed) (r
 
 func (o *otelclient) DoStream(ctx context.Context, cmd rueidis.Completed) (resp rueidis.RedisResultStream) {
 	ctx, span := o.start(ctx, first(cmd.Commands()), sum(cmd.Commands()), o.tAttrs)
+	if o.dbStmtFunc != nil {
+		span.SetAttributes(dbstmt.String(o.dbStmtFunc(cmd.Commands())))
```

Comment:
Hi @chkp-omris, LGTM! But these SetAttributes are not covered by tests

<img width="1219" alt="image" src="https://github.com/redis/rueidis/assets/2727535/6b42a45e-c5b3-458f-ba44-29b4532f7317">

Would you mind to add tests for them?

Reply:
@rueian Sure, I added the option to the unit tests for the client